README.md

```diff
@@ -34,3 +34,55 @@ Run `npm run serve` command for a dev server. Navigate to `http://localhost:8080
 ## Further help
 
 To get more help on the vue CLI use go check out the [Vue-Cli README](https://github.com/vuejs/vue-cli/blob/master/README.md).
+
+## Features and Benefits
+
+### Skeleton loading indicator
+
+The Data Grid shows the Shimmer effect as a loading indicator while fetching data and binding it to the grid during initial rendering, refreshing, or after performing grid actions like sorting, filtering, grouping, and more. The Data Grid offers two types of loading indicator effect, spinner and shimmer.
+
+### Column
+
+Columns define the schema of data source in the Data Grid. It supports formatting, auto generating columns, column definitions, freezing rows and columns, column spanning, text wrapping, column chooser, column menu, and other features.
+
+### Paging
+
+Paging is used to view a segment of data from the assigned data source. The Vue Data Grid offers built-in pager UI with options to customize its entire UI. It also has an on-demand paging mode for effective data retrieval from remote web services.
+
+### Filtering
+
+Filtering helps view particular or related records in the Data Grid, which meet a given criteria. Data grid supports various filter types that include powerful Excel-like filter. Choose appropriate filter type, define your own custom filtering logic, and customize the filtering UI based on the application needs. Diacritic characters can also be filtered.
+
+## Related links
+[Learn More about Vue Data Grid](https://www.syncfusion.com/vue-components/vue-grid?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/vue/confirm?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[Documentation](https://ej2.syncfusion.com/vue/documentation/grid/getting-started?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-vue-data-grid-of-syncfusion?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/vue-components?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/vue?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)
+
+[Online example](https://ej2.syncfusion.com/vue/demos/#/bootstrap5/grid/grid-overview.html?utm_source=github&utm_medium=listing&utm_campaign=vue-grid-github-samples)

```

Comment:
Use default, u can only use overview if there is no default option
https://ej2.syncfusion.com/vue/demos/#/bootstrap5/grid/default.html

Reply:
Done